src/command_modules/azure-cli-container/azure/cli/command_modules/container/tests/latest/test_container_commands.py

```diff
@@ -55,7 +55,8 @@ def test_container_create(self, resource_group, resource_group_location):
         self.cmd('container create -g {rg} -n {container_group_name} --image {image} --os-type {os_type} '
                  '--ip-address {ip_address_type} --dns-name-label {dns_name_label} --ports {ports} --cpu {cpu} --memory {memory} '
                  '--command-line {command} -e {env} --restart-policy {restart_policy} '
-                 '--secrets {secrets} --secrets-mount-path {secrets_mount_path}',
+                 '--secrets {secrets} --secrets-mount-path {secrets_mount_path} '
+                 '--assign --assign-identity',
```

Comment:
Is it possible to add a test for user assigned identities, or at least to unit test your user assigned identities method?

Reply:
That would require creating an actual user assigned identity to run when creating the recordings. Once recorded everything would work fine but I didn't know if it was worth adding the extra dependency for recording the tests. 

Do you think it's something that is worth it?

Reply:
If it would not be too difficult or add too much overhead it would definitely be worth it. That way we can catch any regressions that occur if anyone modifies the method you add. If we do not add this test to the scenario test, we should at least add some unit tests to the method you added for building the identity list.

Reply:
[This](https://github.com/Azure/azure-cli/blob/be891a9360988aacfbc1df1f6853f5d7dc8e3178/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py#L2212-L2224) would be helpful for creating a temporary identity. You can access the identity id by referencing the id field in the json object, which can be retrieved from the execution result by calling get_output_in_json().

Reply:
That's a good solution, Adding it now.